Changelog.md

```diff
@@ -4,6 +4,12 @@
 
 1. **Baseline:** this includes all changes from 5.6.0 and earlier.
 
+1. Experimenting with spdlog
```

Comment:
We need to fix this, but okay for now.

Reply:
ok we can update this when we do the final release